browser/src/map/Map.js

```diff
@@ -1462,7 +1462,9 @@ L.Map = L.Evented.extend({
 			case 'start':
 				if (e.text) {
 					// e.text translated by Core
-					this.showBusy(e.text);
+					this.showBusy(_(e.text));
```

Comment:
Why did this go from e.text to _(e.text) ? Doesn't _() mark it for translation and the comment above suggests that this arrives already translated from code?

Reply:
I have added a new string in kit "Exporting". kit can not translate anything so it needs to be translated in js.

Reply:
You can write
if(e.text === 'Exporting') {
this.showBusy(_('Exporting'));
}
else {
this.showBusy(e.text);
}

It's lame, but this way we can handle translation.

Reply:
used more generic approach, makes code look better and also maybe helpful in future...

show "please wait" if no text is provided.